schemas/book.schema.tpl.json

```diff
@@ -0,0 +1,17 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/Book",
+  "_extends": "creativeWork.schema.tpl.json",
+  "required": [
+      "fullName"
+  ],
+  "properties": {
+    "digitalIdentifier": {
+      "_instruction": "Add the globally unique and persistent digital identifier of this research product. Note that this digital identifier will be used to reference all attached research product versions.",
```

Comment:
should be adjusted since a "book" in the context here is not registered as a research product. 

"Add the globally unique and persistent digital identifier of this book."

Reply:
fixed in next push